### DISCUSSION TITLE
2009/05/20/%E6%8E%A7%E4%BB%B6%E7%9A%84%E9%97%AA%E7%83%81%E5%B1%9E%E6%80%A7

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#263] [Announcements]
# 2009/05/20/%E6%8E%A7%E4%BB%B6%E7%9A%84%E9%97%AA%E7%83%81%E5%B1%9E%E6%80%A7

今天在MSN群上看到有人讨论这个属性，于是自己也研究了一下。以前还真没用过这个属性。

https://blog.qizhen.xyz/2009/05/20/%E6%8E%A7%E4%BB%B6%E7%9A%84%E9%97%AA%E7%83%81%E5%B1%9E%E6%80%A7

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Coral
2009年05月21日 09:37 编辑
NI还真是高效率，LabVIEW已经被汉化成这个程度了。界面看上去挺可爱的。

回复
Qizhen
2009年05月21日 08:24 编辑
上论坛时候经常发现有我没用过的东西:)

回复
杰
2009年05月20日 22:32 编辑
哦。没想到，软虾居然从来没用过。blink属性还是挺好使的，经常用在报警上。我经常还把这个用在状态显示上，譬如说当前测试正在进行，屏蔽了前面板的控件相应，就把某一个控件闪烁着并把caption改成正在测试，告诉用户正在测试，不会理会用户的点击，测试完了，停止闪烁，改其caption为空闲～

回复
